src/model/transaction/moveBlockInContentState.js

```diff
@@ -26,6 +26,24 @@ const invariant = require('invariant');
 
 const {OrderedMap, List} = Immutable;
 
+const transformBlock = (
```

Comment:
Once this PR is merged will then start refactoring our codebase for removing duplication and extracting things like this into utils (I believe this function already exists in a few files) could be also good PR's for contributors